profiles/test_speed_unit.lua

```diff
@@ -1,6 +1,6 @@
 
-local tags = require('lib/tags')
-local Measure = require('lib/measure')
+local tags = require('lib-unidb/tags')
```

Comment:
This file seems a little bit different. Maybe better to duplicate too? How's your idea?

Reply:
> This file seems a little bit different. Maybe better to duplicate too? How's your idea?

No need to duplicate since only unidb has 'speed unit'.
I am thinking a new proposal to create a test-unidb folder that hold all unit tests specific for scripts of unidb. (What exactly gonna be test for scripts. For this one, it seems not necessary) 
Just leave here temporary (but changed the required lib to lib-unidb, otherwise it can not be run).

Any other idea or suggestions?

Reply:
Is it possible to have multiple `test-xxx.lua`? If so, I think it's a better idea. 
If this file is for unidb only(you added in the reverting PR, right?), please move it into `test-unidb/`.

Reply:
> Is it possible to have multiple `test-xxx.lua`? If so, I think it's a better idea.
> If this file is for unidb only(you added in the reverting PR, right?), please move it into `test-unidb/`.

Delete this unnecessary testing file

Reply:
> @hellotechx, I tried to find `lib-unidb` in the feature branch of `feature/revert-pr67` but I couldn't, I could find it on master.
> I think its due to the purpose of this pull is `revert -pr67`, but I think its important each pull request is atomic changes working on valid branch. I think the source of branch `revert-pr67` should from master.

Created a new branch from master to do this job, close this one.

Reply:
Actually you can simply do `git rebase` instead of create another PR.